src/components/views/settings/tabs/user/GeneralUserSettingsTab.js

```diff
@@ -70,7 +70,12 @@ export default class GeneralUserSettingsTab extends React.Component {
         const cli = MatrixClientPeg.get();
 
         const serverSupportsSeparateAddAndBind = await cli.doesServerSupportSeparateAddAndBind();
-        this.setState({serverSupportsSeparateAddAndBind});
+
+        const capabilities = await cli.getCapabilities(); // this is cached
+        const changePasswordCap = capabilities['m.change_password'];
+        const canChangePassword = changePasswordCap && changePasswordCap['enabled'] === false;
```

Comment:
`true` right...?

What happens when the cap is missing for older servers?

Reply:
doh, yes.